extensions/rich_text_components/components_test.py

```diff
@@ -114,6 +114,21 @@ def test_link_validation(self):
         self.check_validation(
             components.Link, valid_items, invalid_items)
 
+    def test_concept_card_validation(self):
+        """Tests concept card component validation."""
+        valid_items = [{
+            'skill_summary-with-value':
+                '{\'id\': \'skill_id\', \'description\': '
+                '\'skill_description\'}',
+        }]
+        invalid_items = [{
+            'skill_summary-with-value': 'javascript:alert(5);',
+            'text-with-value': 'Hello'
+        }]
+
+        self.check_validation(
+            components.Concept, valid_items, invalid_items)
+
```

Comment:
Nit: Move this after `test_collapsible_validation` to keep the alphabetical ordering.

Reply:
Done